### PR TITLE
Mz nightly 20200328

### PR DESCRIPTION
**Pull request by AwesomeKalin**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19596)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Do you know where this application is installed to? I can't seem to find it in Control Panel.

**Comment by AwesomeKalin** [comment]
Program Files

**Comment by OfficialEsco** [comment]
> Do you know where this application is installed to? I can't seem to find it in Control Panel.

This program Extracts its self whereever the exe is executed

@AwesomeKalin
> Program Files
```
*cough* 
[X] Have you tested your manifest locally with winget install --manifest <path>?
```

It extracts itself in `%LocalAppData%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\TempState\WinGet`.

File: manifests/w/WaterbugStudios/MazeratorOld/Nightly 20200328/WaterbugStudios.MazeratorOld.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://download.tuxfamily.org/mazerator/mazerator-old/Nightly%2020200328/Mazerator-Nightly-20200328.exe
+  InstallerSha256: DC511730F6F7779CC04786E1125675621770FA8CA47CA8BD787F75CB1372A01F
+  UpgradeBehavior: install
```
Comment (ItzLevvie) [inline manifests/w/WaterbugStudios/MazeratorOld/Nightly 20200328/WaterbugStudios.MazeratorOld.installer.yaml:18]:
```suggestion
  InstallerSwitches:
    Custom: /D=C:\Program Files
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Requires dependencies to run the application - so I don't think the PR can be accepted:
image

**Comment by AwesomeKalin** [comment]
> Requires dependencies to run the application - so I don't think the PR can be accepted:
> image

Those should be already installed on Windows 10, or there is a file somewhere which installs the dependencies

**Comment by AwesomeKalin** [comment]
Also, I don't know why it failed. There were no warnings or errors

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Need to wait for dependency support to be added.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Adding another "request changes" review for filtering purposes.

**Comment by ItzLevvie** [comment]
> Those should be already installed on Windows 10, or there is a file somewhere which installs the dependencies

All manifests are run against a clean installation of Windows which doesn't have those dependencies installed OOB.

I have accepted your other PRs because @palenshus said that run-time dependencies are allowed; only install-time dependencies aren't allowed.

**Pull request closed without merging**